firmware/src/network_model.c

```diff
@@ -333,14 +290,11 @@ int NetworkModel_SetDHCPMode(const RDMHeader *header,
 
 int NetworkModel_GetZeroconfMode(const RDMHeader *header,
                                  const uint8_t *param_data) {
-  unsigned int index = 0;
```

Comment:
Yeah, that's 5 lines instead of 4. It also relies on overloading the convention of what a negative response size represents.